src/matcha_ml/core/core.py

```diff
@@ -177,7 +183,6 @@ def provision(
 
     with remote_state_manager.use_lock(), remote_state_manager.use_remote_state():
         # create a runner for provisioning resource with Terraform service.
```

Comment:
Move comment to `template_runner = AzureRunner()` or remove comment if it is unnecessary.

Reply:
Removed, thanks.